README.md

```diff
@@ -17,10 +17,10 @@ Scalar Admin pause tool for the Kubernetes environment
                              Namespace that Scalar products you want to pause
                                are deployed. `default` by default.
   -r, --release-name=<helmReleaseName>
-                             Helm's release name that you specify when you run
-                               the `helm install <RELEASE_NAME>` command. You
-                               can see the <RELEASE_NAME> by using the `helm
-                               list` command.
+                             This option is required. Helm's release name that
+                               you specify when you run the `helm install
+                               <RELEASE_NAME>` command. You can see the
+                               <RELEASE_NAME> by using the `helm list` command.
```

Comment:
In this case, since we have one incomplete sentence in between two complete sentences, it feels a little odd. Since the other sentences in the options start as incomplete sentences, how about we start this as follows?:

```suggestion
                             Required. The Helm release name that
                               you specify when you run the `helm install
                               <RELEASE_NAME>` command. You can see the
                               <RELEASE_NAME> by using the `helm list` command.
```

I'm open to other suggestions as well!

Reply:
Ah, I see. That looks good!
I will update it. Thank you for the suggestion!

Reply:
These sentences are defined in the source code side, and we have to apply the code formatter. So, I updated both the code and README in ff667e958e7f87a8c41ce08def1f6a7ab907d774 .
Please take a look again when you have time!